src/atomvm_led_strip.app.src

```diff
@@ -18,7 +18,7 @@
 % SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 %
 
-{application, atomvm_dotstar, [
+{application, atomvm_led_strip, [
```

Comment:
I think you are fairly free to name it whatever you like but the convention is same name on application as directory. You define the start module name in the .app file: {mod, {Module, Args}} . There is no need for one as long as you only have library files and do not need to start any processes (services).

Reply:
This makes perfect sense, thank you for the explanation. I should have actually realized that, since I already updated the example application deps to use atomvm_led_strip.